src/components/TransactionPanel/TransactionPanel.js

```diff
@@ -213,6 +217,73 @@ export class TransactionPanelComponent extends Component {
     const isProviderBanned = isProviderLoaded && currentProvider.attributes.banned;
     const isProviderDeleted = isProviderLoaded && currentProvider.attributes.deleted;
 
+
+    // QR code %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+  const account = 'm';
+  const amount = 'a';
+  const product = 'r';
+  const category = 'cat';
+  const ven = 'ven';
+  const cbu = 'cbu';
+  const web = 'web';
+  const ind = 'ind';
+
+  const venObj = {
+    //[cbu]: canonicalRootURL + '/order/' + values.orderId.uuid + '/details',
+    [cbu]: canonicalRootURL + '/api/accept-privileged?txId=' + currentTransaction.id.uuid,
+    [web]: canonicalRootURL,
+    [ven]: 'Good Dollar Marketplace',
+    [ind]: 'used_for_what'
```

Comment:
put there the uuid/id of the invoice/transaction/receipt
this will be later used to match the blockchain transaction to the receipt, so we will know it was paid

Reply:
Got it